docs/getting_started.mdx

```diff
@@ -8,6 +8,75 @@ tags:
+also [MinIO S3 Instance](https://min.io/docs/minio/container/index.html). By the end of this walkthrough, we will have created some tables on OpenHouse,
+inserted data in them, and queried data. For more information on various docker environments and how to set them up
+please see the [SETUP.md](https://github.com/linkedin/openhouse/blob/main/SETUP.md) guide.
+
+In the consecutive optional section, you can learn more about some simple GRANT REVOKE commands and how
+OpenHouse manages access control.
+
+### Prerequisites
+- [Docker CLI](https://docs.docker.com/get-docker/)
+- [Docker Compose CLI](https://github.com/docker/compose-cli/blob/main/INSTALL.md)
+
+## Create and write to OpenHouse Tables
+### Get environment ready
+First, clone [OpenHouse github repository](https://github.com/linkedin/openhouse) and
+run `./gradlew build` command at the root directory. After the command succeeds you should see `BUILD SUCCESSFUL`
+message.
+
+```shell
+openhouse$main>  ./gradlew build
+```
+
+Execute `docker compose -f infra/recipes/docker-compose/oh-s3-spark/docker-compose.yml up -d --build` command to
+bring up docker containers for OpenHouse services, Spark and S3.
+
+### Run SQL commands
+Let us execute some basic SQL commands to create table, add data and query data.
+
+First login to the driver node and start the spark-shell.
+```shell
+oh-hadoop-spark$main>  docker exec -it local.spark-master /bin/bash
+
+openhouse@0a9ed5853291:/opt/spark$  bin/spark-shell --packages org.apache.iceberg:iceberg-spark-runtime-3.1_2.12:1.2.0,software.amazon.awssdk:bundle:2.20.18,software.amazon.awssdk:url-connection-client:2.20.18   \
+--jars openhouse-spark-runtime_2.12-*-all.jar  \
+--conf spark.sql.extensions=org.apache.iceberg.spark.extensions.IcebergSparkSessionExtensions,com.linkedin.openhouse.spark.extensions.OpenhouseSparkSessionExtensions   \
+--conf spark.sql.catalog.openhouse=org.apache.iceberg.spark.SparkCatalog   \
+--conf spark.sql.catalog.openhouse.catalog-impl=com.linkedin.openhouse.spark.OpenHouseCatalog     \
+--conf spark.sql.catalog.openhouse.io-impl=org.apache.iceberg.aws.s3.S3FileIO   \
+--conf spark.sql.catalog.openhouse.s3.endpoint=http://minioS3:9000  \
+--conf spark.sql.catalog.openhouse.s3.access-key-id=admin  \
+--conf spark.sql.catalog.openhouse.s3.secret-access-key=password  \
+--conf spark.sql.catalog.openhouse.s3.path-style-access=true  \
+--conf spark.sql.catalog.openhouse.metrics-reporter-impl=com.linkedin.openhouse.javaclient.OpenHouseMetricsReporter    \
+--conf spark.sql.catalog.openhouse.uri=http://openhouse-tables:8080   \
+--conf spark.sql.catalog.openhouse.auth-token=$(cat /var/config/openhouse.token) \
+--conf spark.sql.catalog.openhouse.cluster=LocalS3Cluster
+```
+:::note
+the configuration `spark.sql.catalog.openhouse.uri=http://openhouse-tables:8080` points to the docker container
+running the [OpenHouse Catalog Service](./intro.md#catalog-service).
+:::
+:::note
```

Comment:
Also add a note about io-impl pointing to S3FileIO?

Reply:
PTAL